AST-1215 - Added setup prototype for starting adding compatibility for woocommerce elementor widgets

imnavanath opened the pull request:
### Description
- This PR can be base & main PR
- Epic - https://wp-astra.atlassian.net/browse/AST-1215

### Types of changes
- Compatibility for Elementor v3.5.0 

### How has this been tested?
- Elementor & Elementor Pro upcoming beta version

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

Nikschavan commented:
>(DON'T MERGE IT FOR NOW)

@imnavanath - you can convert this to a "Draft" PR until this is ready?

imnavanath commented:
> > (DON'T MERGE IT FOR NOW)
> 
> @imnavanath - you can convert this to a "Draft" PR until this is ready?

Yes I'll keep this in mid now onwards, this PR is ready-to-merge now, As we made this PR as base (branch) PR, so that time I added this line. BTW I've removed that as well from description.
